### PR TITLE
[MM-31085] Create a db init command to initialize the database

### PR DESCRIPTION
**Pull request by mgdelacroix**
#### Summary
Adds a `db init` command to the `mattermost` CLI that loads the configuration and initializes the database for a given DSN

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-31085

#### Release Note
```release-note
Adds a mattermost command to initialize the database
```

### CODE REVIEW

**Review by ethervoid — APPROVED** [review]
LGTM! Great stuff and thank you buddy

File: cmd/mattermost/commands/db.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+		InitDbCmd,
+	)
+
+	RootCmd.AddCommand(
+		DbCmd,
+	)
+}
+
+func initDbCmdF(command *cobra.Command, _ []string) error {
+	dsn := getConfigDSN(command, config.GetEnvironment())
```
Comment (jespino) [inline cmd/mattermost/commands/db.go:50]: Why the database config is mandatory?

**Review by jespino — APPROVED** [review]

**Pull request merged**